site/create.md

```diff
@@ -30,3 +30,24 @@ If the model or variables file contains encrypted passwords, add the `-use_encry
 ### Using Multiple Models
 
 The Create Domain Tool supports the use of multiple models, as described in [Using Multiple Models](../README.md#using-multiple-models).
+
+### Development Domain and "boot.properties"
+
+When creating a development domain, WDT provides the convenience of creating a boot.properties for each of the servers in the domain. The boot.properties will contain encrypted values of the Admin Server username and password. When the Admin server or Managed server is started, WebLogic Server will bypass the prompt for credentials, instead using the credentials from the boot.properties.
+
+A domain is considered production mode if the ServerStartMode option is set to 'prod' and/or the domain ProductionModeEnabled is set to true. The default value for both of these attributes is development.
```

Comment:
A domain is considered production mode if the ServerStartMode option is set to 'prod' and/or the domain ProductionModeEnabled is set to true. -> A domain is in production mode if the `ServerStartMode` option is set to `prod` or the domain `ProductionModeEnabled` option is set to `true`.  (Use back ticks to render code font)
development. -> development mode.

Reply:
Done